CONTRIBUTING.md

```diff
@@ -1,15 +1,50 @@
+---
+layout: post
+title: Contributing
+permalink: /contributing/
+---
+
 Welcome to Open Source Design
 =============================
 
+Welcome! 
+
+Before diving into Open Source Design, have a look at our mission: 
+
+> We are a **community** of **designers and developers** pushing more open design processes and improving the design of **open source** apps.
+
+There are a couple of important words there to keep in mind before exploring Open Source Design. 
+
+* Community: Open Source Design is first and foremost a community of people. We hang out in IRC, we chat in our forum, and we work on projects together on GitHub. There is a very loose hierarchy, but it's largely a based on how long you've been involved in the group. People who have been here longer know how the community works and how it operates. If you want to learn more about the rules we have formulated over time, check out our [by-laws](http://opensourcedesign.net/by-laws/) and our [code of conduct](opensourcedesign.net/by-laws/).
```

Comment:
By laws and code go to the same place? The second link is also missing the protocol.